configuration.deploy.yaml

```diff
@@ -12,5 +12,6 @@ session_parameters:
     timeout: 86400
 superadmins:
 - superadmin
+jwt_key: p3JcQcTwcnKxr39t7p6B7cbaE8CvuPPoOj3kE4HyoCrlKM10BLly72X3Q90uJ8kaDb
```

Comment:
Can you add a comment on how to generate this string so that people could replace it easily on setup ?

The key should also be replicated with small explanation in `INGInious/docker-compose.yml`

As I guess this is a mandatory parameter, we should also update the installer script to generate it appropriately in `INGInious/inginious/frontend/installer.py`.